src/lib/inbox/inbox.ts

```diff
@@ -42,12 +45,12 @@ type ForceInclusionParams = FetchedEvent<MessageDeliveredEvent> & {
   delayedAcc: string
 }
 
-type GasComponentsWithL2Part = {
+type GasComponentsWithChainPart = {
   gasEstimate: BigNumber
   gasEstimateForL1: BigNumber
```

Comment:
```suggestion
  gasEstimateForParentChain: BigNumber
```

Reply:
yeh, agree, prob best to have the same name as nodeinterface